src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -243,25 +243,25 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
           isLoading={isEventsLoading}
           {...scheduleNGWithAttendees}
         />
-      </div>
-      {!isEventsLoading && (
-        <div className="NavBarWrapper__download-buttons">
-          {isShowPersonalDownloadBtn && (
+        {!isEventsLoading && (
+          <div className="NavBarSchedule__downloadButtons">
+            {isShowPersonalDownloadBtn && (
+              <Button
+                onClick={downloadPersonalEventsCalendar}
+                customClass="NavBarSchedule__downloadScheduleBtn"
+              >
+                Download your schedule
+              </Button>
```

Comment:
Possible replacement `ButtonNG` due to getting rid of styled components if it is no trouble for the PR.

Reply:
fixed in 2c16847b0928ba957a737453f7b445db887dda23